skythought/tools/inference_and_check.py

```diff
@@ -305,7 +305,11 @@ def main():
     args = parser.parse_args()
     
     handler: TaskHandler = TASK_HANDLERS[args.dataset]()
-    temperatures = [1] if args.model.startswith("openai/o1") else args.temperatures
+    temperatures = [1] if args.model.startswith("openai/o1") else args.temperatures 
+    if args.model.startswith("NovaSky-AI/Sky-T1-32B-Preview") and ("AIME" in args.dataset or "GPQADiamond" in args.dataset):
```

Comment:
can this not being hardcoded? we can ask the user to pass in temperature outside